core/src/main/scala/scalaz/std/Option.scala

```diff
@@ -7,9 +7,13 @@ sealed trait OptionInstances0 {
   }
 
   implicit def optionBand[A: Band]: Band[Option[A]] =
-    new OptionMonoid[A] with Band[Option[A]] {
+    new OptionSemigroup[A] with Band[Option[A]] {
       override def B = implicitly
     }
+  implicit def optionSemiLattice[A: SemiLattice]: SemiLattice[Option[A]] =
```

Comment:
I think this will be ambiguous with the above, right? Need to put them in the hierarchy properly.